lib/go-tc/server_capabilities.go

```diff
@@ -28,6 +28,7 @@ type ServerCapabilitiesResponse struct {
 // ServerCapability contains information about a given ServerCapability in Traffic Ops.
 type ServerCapability struct {
 	Name        string     `json:"name" db:"name"`
+	Description string     `json:"description" db:"description"`
```

Comment:
This change should result in a new Server Capability structure instead of modifying the old one. Using this structure in old API versions will result in them having a `description` property, which they can't have.